src/System.Private.Xml/src/System/Xml/Base64Encoder.cs

```diff
@@ -13,7 +13,7 @@ internal abstract partial class Base64Encoder
         private int _leftOverBytesCount;
         private char[] _charsLine;
 
-        internal const int Base64LineSize = 76;
+        internal const int Base64LineSize = 1024;
```

Comment:
This is a won't fix as rfc2045 only applies to mime encoded base64. If this was changing where new lines are being emitted, it would break existing unit tests.